### PR TITLE
Added TextBlockStyles and refurbished AboutPage

### PR DESCRIPTION
**Pull request by Hofkee**

### CODE REVIEW

**Comment by Hofkee** [comment]
[Create a base textblock style](https://trello.com/c/QFf1wfkd/52-create-a-base-textblock-style)

File: ToNote/Views/AboutPageView.xaml [hunk 1]
```diff
@@ -5,32 +5,33 @@
...
 
-        <StackPanel Grid.Row="0">
-            <TextBlock Text="Developers: " HorizontalAlignment="Center" Margin="0,10,0,0" FontWeight="Bold" />
-        </StackPanel>
         <Grid Grid.Row="1">
-            <Image Height="100" Width="100" HorizontalAlignment="Left" Margin="140,0,0,0" Stretch="UniformToFill" Source="https://media.npr.org/assets/img/2016/01/07/macaca_nigra_self-portrait_custom-a8e13582c9ca6f71f5cd62815b8bb5d6ff112dc2-s800-c85.jpg"/>
-            <TextBlock Text="Arijus Treinys" HorizontalAlignment="Left" Margin="140,0,00,0" VerticalAlignment="Bottom" />
-            <Image Height="100" Width="100" HorizontalAlignment="Right" Margin="0,0,140,0" Stretch="UniformToFill" Source="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQrAzB3dynTfZ4CioA56_XksdHsXMZUZgv4HfSb5O9js5BBjEix" />
-            <TextBlock Text="Audrius Liuberskis" HorizontalAlignment="Right" Margin="00,00,140,0" VerticalAlignment="Bottom" />
+            <TextBlock Text="Version" FontSize="14" Margin="15,-40,0,0" Foreground="#404040"/>
```
Comment (t-eliks) [inline ToNote/Views/AboutPageView.xaml]: Dark foreground on dark background

File: ToNote/Views/AboutPageView.xaml [hunk 2]
```diff
@@ -5,32 +5,33 @@
...
 
+
         <Grid Grid.Row="2">
-            <Image Height="100" Width="100" HorizontalAlignment="Left" Margin="40,0,0,0" Stretch="UniformToFill" Source="https://www.kontrowersje.net/sites/default/files/chimpanzee-photo-640x511.jpg" />
-            <TextBlock Text="Donatas Kuprys" HorizontalAlignment="Left" Margin="40,00,00,0" VerticalAlignment="Bottom" />
-            <Image Height="100" Width="100" HorizontalAlignment="Center" Margin="0,0,00,0" Stretch="Fill" Source="https://s3.amazonaws.com/spectrumnews-web-assets/wp-content/uploads/2018/11/13154625/20181112-SHANK3monkey-844.jpg"/>
-            <TextBlock Text="Eligijus Jocius" HorizontalAlignment="Center" Margin="00,00,00,0" VerticalAlignment="Bottom" />
-            <Image Height="100" Width="100" HorizontalAlignment="Right" Margin="0,0,40,0" Stretch="Fill" Source="https://www.wildfutures.org/wp/wp-content/uploads/2009/09/BANJO-LM3.jpg"/>
-            <TextBlock Text="Rokas Gulbinas" HorizontalAlignment="Right" Margin="00,00,40,0" VerticalAlignment="Bottom" />
+            <TextBlock Text="Developers" Grid.Row="1" VerticalAlignment="Bottom" Margin="20,0,20,85" FontSize="14" Foreground="#404040" />
```
Comment (t-eliks) [inline ToNote/Views/AboutPageView.xaml]: Also here

File: ToNote/Views/AboutPageView.xaml (hunk 1 above)
Comment (Hofkee) [reply ToNote/Views/AboutPageView.xaml]: IMO it looks fine

**Comment by Kodnot** [comment]
1. Make the "ToNote for Windows" centered.
1. Move "Developers" Section higher. It should go directly after "Version". The "2019 Team Sugma" should be moved to the bottom, it should be like a footer (and should be spelled like I spelt it here, with a space and without a dot at the end. Add a copyright sign for bonus points)
1. Fix indentation. Now Developers are centered, while Version is stuck to the left. Probably remove indentation and centering from the development section.
1. Remove dots after developer list

**Review by Kodnot — CHANGES_REQUESTED** [review]
1. Make the "ToNote for Windows" centered.
1. Move "Developers" Section higher. It should go directly after "Version". The "2019 Team Sugma" should be moved to the bottom, it should be like a footer (and should be spelled like I spelt it here, with a space and without a dot at the end. Add a copyright sign for bonus points)
1. Fix indentation. Now Developers are centered, while Version is stuck to the left. Probably remove indentation and centering from the development section.
1. Remove dots after developer list

**Review by Kodnot — APPROVED** [review]

**Review by Kodnot — DISMISSED** [review]
The developer list is still sliiightly indented. Also, the window breaks if resized. @t-eliks , would it be difficult to lock the about page size so that it cannot be resized?

**Comment by t-eliks** [comment]
> The developer list is still sliiightly indented. Also, the window breaks if resized. @t-eliks , would it be difficult to lock the about page size so that it cannot be resized?

Need to add a boolean property to BaseDialogViewModel and bind ResizeMode with a converter to it. Shouldn't use the ResizeMode enumerator to uphold the philosophy of ViewModel doesn't know anything about the View.

**Comment by Hofkee** [comment]
Visur marginai sudeti po 15 is kaires nezinau kodel atrodo, kad siek tiek iseje

**Review by Kodnot — APPROVED** [review]

**Pull request merged**